force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -347,22 +452,27 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static List<String> getHighPaidEmployees() {
         // Initialize a Map
-        Map<String, Integer> employeeSalaries = new Map<String, Integer>();
-
+        Map<String, Integer> employeeSalaries = new Map<String, Integer>{
         // Add key-value pairs to the Map
-        // add John Doe => 50000
-        // add Jane Smith => 60000
-        // add Sam Brown => 55000
-        // add Alice Johnson => 65000
-
+         'John Doe' => 50000,
+         'Jane Smith' => 60000,
+         'Sam Brown' => 55000,
+         'Alice Johnson' => 65000
+        };
+        String[] highPaidEmployees =  new String[]{};
         // Initialize a list to store the names of high paid employees
-
+            for (String i : employeeSalaries.keySet()){
+                Integer dollaz = employeeSalaries.get(i);
+                if (dollaz>55000){
+                    highPaidEmployees.add(i);
+                }
+            }
```

Comment:
I would use a value like name rather than i  in your loop. The keyset is names. It makes it easier to follow.  Small thing.  I like to name maps value By key. So this map would  salaryByName